vumi/message.py

```diff
@@ -429,3 +429,24 @@ def validate_fields(self):
             self.assert_field_present(extra_field)
             if not check(self[extra_field]):
                 raise InvalidMessageField(extra_field)
+
+
+class TransportStatus(TransportMessage):
+    """Message about a status event emitted by a transport.
+    """
```

Comment:
It would be ideal if we could subclass `Message` instead (so that non-transport worker types could also publshing status events), but I think we need it to be subclass of `TransportMessage` for it to work with connectors.